frontend/templates/product-list/sections/FilterableProductsSection/facets/drawer/index.tsx

```diff
@@ -218,7 +218,7 @@ type ClearAllButtonProps = {
 };
 
 function ClearAllButton({ isVisible }: ClearAllButtonProps) {
-   const { refine } = useClearRefinements();
+   const { refine } = useClearRefinements({});
```

Comment:
Is there a reason why we now have to pass a parameter? was it broken before?

Reply:
No, it is just a typo that has no impact on functionality. 
If no property is overridden the hook sticks with default values.

I will clear it out to avoid any doubt